src/ViewModel/Converter/DigestSocialMediaSharersConverter.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace eLife\Journal\ViewModel\Converter;
+
+use eLife\ApiSdk\Model\Digest;
+use eLife\Journal\Helper\LicenceUri;
```

Comment:
This use statement is not needed. We should really be picking this up in ci. Could be a followup task. This is not blocking release but may as well be removed as we need to add a test anyway.

Reply:
The use statement is removed.